### DISCUSSION TITLE
Published code lists not appearing on OpenCodelists

### DISCUSSION THREAD

**Discussion by hjforbes** [opensafely/documentation#480] [Codelists] [answered]
Hi

I have added five diabetes code lists to OpenCodelists, and published them. For example gestational diabetes (link from the my codelists section: https://www.opencodelists.org/codelist/user/hjforbes/gestational-diabetes/1ed423d1/)

None are appearing in the OpenCodelists list however, when I search for them.  What do I need to do for them to appear?

Thanks
Harriet

**Comment by brianmackenna** [comment, marked as answer]
HI @hjforbes . A few thing

1. You can use them in your current location if needed
1. Only codelists attached to a few organisations currently appear in search
1. You can get them added to the OpenSAFELY org if you have followed the [process for signing off](https://docs.opensafely.org/codelist-creation/) (i.e. 2 reviews etc) or if you are working with a new organisation we can create an account for there (@inglesp  can help with either of these)

Separately just wondering why you are using CTV3 and not snomed? We have a codelist from NHS Digital for [gestational diabetes on there](https://www.opencodelists.org/codelist/nhsd-primary-care-domain-refsets/gestdiab_cod/20200812/). Might be more useful ones amongst the [NHSD refsets](https://www.opencodelists.org/codelist/nhsd-primary-care-domain-refsets/)

**Reply by hjforbes** [reply]
Thank you. 

I haven't developed these lists - I am merely uploading them.  As such I'm not sure why CTV3 codes, rather than SNOMED, were used, but I'm guessing it's because they had a CTV3 code list available. 

Really great to see the NHSD refsets - I didn't know they existed!  I will look through those for other useful code lists. Thanks
